cudamapper/include/claragenomics/cudamapper/overlapper.hpp

```diff
@@ -52,6 +48,14 @@ class Overlapper
                                 const std::vector<Overlap>& overlaps,
                                 size_t min_residues    = 20,
                                 size_t min_overlap_len = 50);
+
+    /// \brief updates read names for vector of overlaps output from get_overlaps
+    /// \param overlaps input vector of overlaps generated in get_overlaps
+    /// \param index_query
+    /// \param index_target
+    static void update_read_names(std::vector<Overlap>& overlaps,
```

Comment:
Is this function the same for all possible `overlapper` implementations? I think it is, but want to make sure I understand it correctly (otherwise implementation should go to `OverlapperTriggered`)

Reply:
I assumed it will be the same.